Extension/src/LanguageServer/settings.ts

```diff
@@ -517,99 +508,58 @@ export class CppSettings extends Settings {
+    public get doxygenSectionTags(): string[] { return this.getAsArrayOfStrings("doxygen.sectionTags"); }
+    public get hover(): string { return this.getAsString("hover"); }
+    public get markdownInComments(): string { return this.getAsString("markdownInComments"); }
+    public get legacyCompilerArgsBehavior(): boolean { return this.getAsBoolean("legacyCompilerArgsBehavior"); }
     public get inlayHintsAutoDeclarationTypes(): boolean { return this.getAsBoolean("inlayHints.autoDeclarationTypes.enabled"); }
     public get inlayHintsAutoDeclarationTypesShowOnLeft(): boolean { return this.getAsBoolean("inlayHints.autoDeclarationTypes.showOnLeft"); }
     public get inlayHintsParameterNames(): boolean { return this.getAsBoolean("inlayHints.parameterNames.enabled"); }
     public get inlayHintsParameterNamesSuppressName(): boolean { return this.getAsBoolean("inlayHints.parameterNames.suppressWhenArgumentContainsName"); }
     public get inlayHintsParameterNamesHideLeadingUnderscores(): boolean { return this.getAsBoolean("inlayHints.parameterNames.hideLeadingUnderscores"); }
     public get inlayHintsReferenceOperator(): boolean { return this.getAsBoolean("inlayHints.referenceOperator.enabled"); }
     public get inlayHintsReferenceOperatorShowSpace(): boolean { return this.getAsBoolean("inlayHints.referenceOperator.showSpace"); }
-    public get enhancedColorization(): boolean { return this.getAsString("enhancedColorization") === "enabled" && this.intelliSenseEngine === "default" && vscode.workspace.getConfiguration("workbench").get<string>("colorTheme") !== "Default High Contrast"; }
-    public get formattingEngine(): string | undefined { return this.getAsString("formatting"); }
+    public get isEnhancedColorizationEnabled(): boolean { return this.getAsString("enhancedColorization") === "enabled" && this.intelliSenseEngine === "default" && vscode.workspace.getConfiguration("workbench").get<string>("colorTheme") !== "Default High Contrast"; }
+    public get formattingEngine(): string { return this.getAsString("formatting"); }
     public get vcFormatIndentBraces(): boolean { return this.getAsBoolean("vcFormat.indent.braces"); }
-    public get vcFormatIndentMultiLineRelativeTo(): string {
-        // These strings have default values in package.json, so should never be undefined.
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        return this.getAsString("vcFormat.indent.multiLineRelativeTo")!;
-    }
-    public get vcFormatIndentWithinParentheses(): string {
-        // These strings have default values in package.json, so should never be undefined.
-        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-        return this.getAsString("vcFormat.indent.withinParentheses")!;
-    }
+    public get vcFormatIndentMultiLineRelativeTo(): string {return this.getAsString("vcFormat.indent.multiLineRelativeTo");}
```

Comment:
A lot of these function bodies have the wrong spacing.  Can you please fix them? There should be a space after `{` and before `}`

```
{ return this.getAsString("vcFormat.indent.multiLineRelativeTo"); }
```

Reply:
I think you should be able to run the formatter.